dissect/target/plugins/apps/remoteaccess/teamviewer.py

```diff
@@ -101,11 +113,11 @@ def logs(self):
 
                     timestamp = datetime.strptime(f"{ts_day} {ts_time}", "%Y/%m/%d %H:%M:%S")
 
-                    yield RemoteAccessRecord(
-                        tool="teamviewer",
+                    yield self.RemoteAccessLogRecord(
                         ts=timestamp,
-                        logfile=str(logfile),
-                        description=description,
+                        tool="teamviewer",
+                        message=message,
+                        source=logfile,
+                        _user=user_details.user if user_details else None,
                         _target=self.target,
```

Comment:
```suggestion
                        _target=self.target,
                        _user=user_details.user if user_details else None,
```
Consistency in order.